candle-transformers/src/models/t5.rs

```diff
@@ -653,12 +677,19 @@ impl T5ForConditionalGeneration {
         let decoder_output = self
             .decoder
             .forward(decoder_input_ids, Some(encoder_output))?;
-        let sequence_output = decoder_output
+        let mut sequence_output = decoder_output
```

Comment:
Avoid using `mut` here as it's easy to do a `let ... = if self.tie_word_embeddings {} else {};` instead.

Reply:
Done. PTAL.